go/tasks/pluginmachinery/core/exec_context.go

```diff
@@ -57,6 +59,9 @@ type TaskExecutionContext interface {
 
 	// Returns a handle to the Task events recorder, which get stored in the Admin.
 	EventsRecorder() EventsRecorder
+
+	// Returns details about any resources allocated during the execution of this task.
+	GetResourcePoolInfo() []*event.ResourcePoolInfo
```

Comment:
This is not used by plugins, right? is it needed here?

Reply:
sure, it made things a little cleaner in propeller to be able to just the interface type but updated. ptal @EngHabu